### PR TITLE
Environments list page

### PR DESCRIPTION
**Pull request by leahjlou**
Start of environments list page, showing latest deploy

Screen Shot 2022-07-11 at 6 19 06 PM

### CODE REVIEW

File: frontend/Console/Environments/EnvironmentsList.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      return fuseResult.map((r) => r.item);
+    }
+  }, [environments, search]);
+
+  return (
+    <MainContent>
+      <PageHeader>Environments List</PageHeader>
+      <PageExplanation
+        docsLink="/docs/concepts/environments"
+        briefExplanation="Environments are standalone, self-contained instances of your web application. Many teams use environments called “development”, “staging”, and “production”."
```
Comment (joeldenning) [inline frontend/Console/Environments/EnvironmentsList.tsx:41]: You're using the curly quotation marks around development, staging, and production - is that on purpose? I generally only use those accidentally when copying from Notes app on Mac and then regret doing so. If it's on purpose, that's okay, just checking

**Review by joeldenning — COMMENTED** [review]
Looks good a few minor comments

File: frontend/Console/Environments/EnvironmentsList.tsx (hunk 1 above)
Comment (leahjlou) [reply frontend/Console/Environments/EnvironmentsList.tsx:41]: Yeah I'm doing it on purpose for display purposes, but I don't have strong feelings about it, I just think they look a little bit nicer in display

Comment (leahjlou) [reply frontend/Console/Environments/EnvironmentsList.tsx]: I ended up changing these to italics, I don't know if it's better but it looks a little less noisy.

**Pull request merged**